javascript/src/test/pages/Schedule/PersonalSchedule.test.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import PersonalSchedule from "main/pages/Schedule/PersonalSchedule";
+import { useAuth0 } from "@auth0/auth0-react";
+import userEvent from "@testing-library/user-event";
+import useSWR from "swr";
+import { waitFor } from "@testing-library/react";
+import { useToasts } from "react-toast-notifications";
+import { useParams } from "react-router-dom";

```

Comment:
If you eliminate PersonalScheduleTable.js (since it's redundant with ScheduleTable.js) this test file can go away as well.